pkg/users/users.js

```diff
@@ -82,7 +82,7 @@ function AccountsPage() {
         const handleShadow = cockpit.file("/etc/shadow", { superuser: "try" });
         handleShadow.watch(() => debouncedGetLogins(), { read: false });
 
-        const handleLogindef = cockpit.file("/etc/login.defs", { superuser: true });
```

Comment:
This is a functionality change, and a dubious one at that.  Why do we need to be root at all for this?

I'd either blindly rewrite this to require, or deal with it in a separate PR with a good commit message about why we're making the change (and in that case, I'd suggest dropping the superuser flag alltogether).

Reply:
It should not need root at all, and has been reworked in https://github.com/cockpit-project/cockpit/pull/20309

I am happy about dropping this ambiguous stuff now